test/mocks/v1/testing_snapshot_emitter_test.go

```diff
@@ -1,5 +1,3 @@
-// Code generated by solo-kit. DO NOT EDIT.
-
```

Comment:
why did this get removed? i think we would want this to stay yes?

Reply:
if it were generated CI would have failed...